packages/fxa-payments-server/src/components/Header/index.tsx

```diff
@@ -1,30 +1,41 @@
 import React from 'react';
-
-import './index.scss';
 import { Profile } from '../../store/types';
+import firefoxLogo from '../../images/firefox-logo-combined.svg';
+import { useLocalization } from '@fluent/react';
 
-type HeaderProps = {
+export type HeaderProps = {
   profile?: Profile;
   className?: string;
 };
 
 export const Header = ({ profile, className = 'default' }: HeaderProps) => {
+  const { l10n } = useLocalization();
   let profileSection = null;
+
   if (profile) {
     const { avatar, displayName, email } = profile;
     profileSection = (
       <img
-        className="avatar hoisted"
+        className="rounded-full w-8 h-8"
         src={avatar}
         data-testid="avatar"
         alt={displayName || email}
+        title={displayName || email}
       />
     );
   }
 
   return (
-    <header className={`${className} fxa-settings-header`} role="banner">
-      <div className="fxa-logo" data-testid="branding" title="firefox"></div>
+    <header
+      className={`${className} flex justify-between items-center bg-white shadow h-16 fixed left-0 top-0 my-0 mx-auto px-7 py-4 w-full z-10 tablet:h-20`}
```

Comment:
px-4 py-0?

Reply:
Fixed!

I originally changed it to try and match more closely with `fxa-settings` (and using `px-7` instead of `px-8` to line up with the checkout containers in mobile), but since it will be revised in FXA-6233, agreed that it should be kept as is at `px-4 py-0` until then.